contrib/gitian-descriptors/gitian-osx-signer.yml

```diff
@@ -12,6 +12,7 @@ remotes:
   "dir": "signature"
 files:
 - "bitcoin-osx-unsigned.tar.gz"
+- "signature-osx.tar.gz"
```

Comment:
How do we not provide this file? AFAIK anything listed here is required by gitian?

Reply:
AFAIK it only produces a warning (`sha256sum: signature-osx.tar.gz: No such file or directory`).